pkg/execute/executor.go

```diff
@@ -111,16 +122,32 @@ func (action NotifierAction) String() string {
 	return string(action)
 }
 
+// ProfileAction creates custom type for notifier actions
+type ProfileAction string
+
+// Defines constants for notifier actions
+const (
+	ProfileGet        ProfileAction = "get"
+	ProfileList       ProfileAction = "list"
+	ProfileShowConfig ProfileAction = "showconfig"
+)
```

Comment:
Please move this to the `const` group already declared

Reply:
I think we have grouped similar const in one. Let's use the same pattern then. Please ignore this comment.